lib/client.js

```diff
@@ -41,6 +41,7 @@ var Client = function Client(dsn, options) {
   this.root = options.root || process.cwd();
   this.transport = options.transport || transports[this.dsn.protocol];
   this.release = options.release || process.env.SENTRY_RELEASE || '';
+  this.environment = options.environment || process.env.SENTRY_ENVIRONMENT || '';
```

Comment:
@mattrobenolt @benvinegar I thought about that but saw from the blog:

> By default we automatically select the most production like environment

and I also saw [this](https://github.com/getsentry/raven-node/blob/aa6ac5082272d6dd1f41f8e0231fc754adb72974/History.md#050---482013) so I opted not too. I'm happy to add it in as a fallback!

fwiw, most people I know run node in 'production' for staging sites so I don't know how much value it would add